contracts/tgrade-valset/src/error.rs

```diff
@@ -71,6 +68,13 @@ pub enum ContractError {
 
     #[error("Jail did not yet expire")]
     JailDidNotExpire {},
+
+    #[error("Invalid metadata - {data} length must be {min}-{max} characters")]
+    InvalidMetadata {
+        data: String,
```

Comment:
nit, you could use `&'static str` here, right?
(not needed change)

Reply:
I could, good point